lib/iex/lib/iex/helpers.ex

```diff
@@ -643,4 +644,42 @@ defmodule IEx.Helpers do
              Integer.to_char_list(z) ++ '>'
     )
   end
+
+  @doc """
+  Deloys a given module's beam code to a list of connected nodes.
+
+  This function is useful for development and debugging when you have code that
+  has been compiled or updated locally that you want to run on other nodes.
+  
+  The node list defaults to a list of all connected nodes.
+
+  Returns `{:error, :nofile}` if the object code (i.e. ".beam" file) for the module
+  could not be found locally.
+
+  ## Examples
+
+      nl(HelloWorld)
+      #=> {:ok, [{:node1@easthost, :loaded, HelloWorld},
```

Comment:
As @josevalim said, current output is totally fine.
I named mine as _actual_ since this is what you see as `IO.inspect/2` output with default options (`width: 80`).